packages/ng/skeleton/skeleton-index-table/skeleton-index-table.component.html

```diff
@@ -0,0 +1,36 @@
+@if (tableBodyOnly) {
+<tbody class="indexTable-body">
+	@for (row of rowsNumber(); track row) {
+	<tr class="indexTable-body-row">
+		@for (col of colsNumber(); track col) {
+		<td class="indexTable-body-row-cell">
+			<span class="skeleton-item" [style.--components-skeleton-text-width]="getRandomPercent()"></span>
+		</td>
+		}
+	</tr>
+	}
+</tbody>
+} @else {
+<table class="indexTable mod-layoutFixed skeleton is-loading" inert="inert">
+	<thead class="indexTable-head">
+		<tr class="indexTable-head-row">
+			@for (col of colsNumber(); track col) {
+			<th class="indexTable-head-row-cell" scope="col">
+				<span class="skeleton-item mod-dark" [style.--components-skeleton-text-width]="getRandomPercent()"></span>
+			</th>
+			}
+		</tr>
+	</thead>
+	<tbody class="indexTable-body">
+		@for (row of rowsNumber(); track row) {
+		<tr class="indexTable-body-row">
+			@for (col of colsNumber(); track col) {
+			<td class="indexTable-body-row-cell">
+				<span class="skeleton-item" [style.--components-skeleton-text-width]="getRandomPercent()"></span>
+			</td>
+			}
+		</tr>
+		}
+	</tbody>
+</table>
+}
```

Comment:
To avoid repeating the same code, could the logic be inverted?

```html
@if (!tableBodyOnly) {
<table class="table skeleton is-loading" inert="inert">
	<thead class="table-head">
		<tr class="table-head-row">
			@for (col of colsNumber(); track col) {
			<th class="table-head-row-cell">
				<span class="skeleton-item mod-dark u-textS" [style.--components-skeleton-text-width]="getRandomPercent()"></span>
			</th>
			}
		</tr>
	</thead>
}

<tbody class="table-body">
	@for (row of rowsNumber(); track row) {
	<tr class="table-body-row">
		@for (col of colsNumber(); track col) {
		<td class="table-body-row-cell">
			<span class="skeleton-item" [style.--components-skeleton-text-width]="getRandomPercent()"></span>
		</td>
		}
	</tr>
	}
</tbody>

@if (!tableBodyOnly) {
</table>
}
```

Reply:
No, because the wrapper (`table`) is not present either.

Reply:
An `ng-template` was the solution, thank you!